spec/integration/show_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'capybara/rails'
+require 'capybara/minitest'
+
```

Comment:
- [ ] When I run the tests file for the `User#Show` page, I get this error:

![image](https://github.com/Kaghenimbale/blog-app/assets/77106962/e58bd942-e728-48f8-9b04-fc8418f888da)

You ought to require the rails_helper and not the dependencies. I would advise you to remove the other dependencies you are requiring. 👍 😎